pkgs/tools/text/cmigemo/default.nix

```diff
@@ -23,7 +23,27 @@ stdenv.mkDerivation {
     cp ${skk-dicts}/share/SKK-JISYO.L source/dict/
   '';
 
-  patches = [ ./no-http-tool-check.patch ];
+  patches = [
+    ./no-http-tool-check.patch
+
+    # Pending upstream inclusion patches to support parallel build and install:
+    #   https://github.com/koron/cmigemo/pull/29
```

Comment:
The pull request was merged. So updating rev of `fetchFromGitHub` is enough?

Reply:
Good point! Dropped separate path fetching and updated to latest `master`. While at it dropped unused `fetchurl` parameter.

Reply:
Thank you!